### PR TITLE
Add initial clipboard implementation

### PR DESCRIPTION
**Pull request by msisov**

### CODE REVIEW

File: ui/base/clipboard/clipboard.cc [hunk 1]
```diff
@@ -91,6 +91,8 @@ base::Time Clipboard::GetLastModifiedTime() const {
 
 void Clipboard::ClearLastModifiedTime() {}
 
+void Clipboard::SetDelegate(ClipboardDelegate* delegate) {}
```
Comment (jkim-julie) [inline ui/base/clipboard/clipboard.cc:95]: #if defined(USE_OZONE) ?

**Comment by ammen99** [comment]
Hi, why was this closed? Is wayland clipboard support merged upstream already?

**Comment by jkim-julie** [comment]
Hi @ammen99, we're considering how to implement it generally like below.
1) Utilize previously merged Ozone level clipboard APIs,
2) Design and implement general clipboard handling on the aura framework level.
Due to priorities of tasks we're having, it's not handled at this moment but we're planning to do.
Thanks for being interested in it.

**Comment by msisov** [comment]
@ammen99, the clipboard support was merged in https://github.com/Igalia/chromium/pull/506

**Comment by ammen99** [comment]
@msisov that's great news, I'll start rebuilding (hopefully I can use wayland native as daily driver, last time it was quite stable with only clipboard missing). Should I still build from this fork or from upstream master?

**Pull request closed without merging**